lib/kubernetes-deploy/options_helper.rb

```diff
@@ -6,9 +6,9 @@ class OptionsError < StandardError; end
 
     STDIN_TEMP_FILE = "from_stdin.yml.erb"
     class << self
-      def with_processed_template_paths(template_paths)
+      def with_processed_template_paths(template_paths, require_explicit_path: false)
```

Comment:
I think we need to add a test showing we produce a nice error for the new flag..